openfasoc/generators/temp-sense-gen/flow/util/openfasoc/cdl_parser.py

```diff
@@ -48,9 +48,17 @@
     filedata = filedata.replace(
         "VIN ", "", 1
     )  # replace VIN with nothing one time (i.e. delete the pin in toplevel)
-    filedata = filedata.replace(
-        "r_VIN", "", 1
-    )  # replace r_VIN with nothing one time (i.e. delete the pin in toplevel)
+
+    if int(os.environ["VIN_ROUTE_CONNECTION_POINTS"]) > 1:
```

Comment:
Can you explain a little more what you are doing here?

Reply:
Of course. So, this new script `create_routable_power_net.tcl` will create a routable version of the VIN net, called r_VIN. This net is used to connect the header cells to the VIN voltage domain. The user may want to use more than one pin to make this connection, to lower the route's resistance.

So, in case he specifies to do so, the `create_routable_power_net.tcl` script creates pins called `r_VIN(0)`, `r_VIN(1)`, `r_VIN(2)` and so on, By default in OpenROAD, though, they appear in the top subckt of the CDL file, so they have to be removed for LVS to pass.

If the user wants the connection to be created with 1 pin only, then the pin is called simply r_VIN (automatically by OpenROAD), the same name of the net, and also has to be removed from the CDL file for LVS to pass (this is what was implemented before this PR).

Reply:
I'll update the comments in the code :)